app/views/shared/_recent_activity.html.erb

```diff
@@ -7,6 +7,32 @@
 </h2>
 
 <ul class="recent_activity unstyled_list">
+
+	<% if @current_user && @current_user.last_url %>
```

Comment:
fyi: there is some weird spacing stuff going on in this section.  E.g. Pick up where you left off and the actual link are spaced like halfway across the screen.